1038-vertical-nav-design-guidance.md

```diff
@@ -0,0 +1,35 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-21
+---
+
+# Vertical Nav Design Guidance
+
+The [vertical nav](https://clarity.design/documentation/vertical-nav) component provides vertically-aligned navigation links. It can fit as many navigation links as needed by scrolling when the content exceeds the viewport.
+
+
+## Guidance
+
+- **Must** use to collapse and expand the navigation to free up space for the content area.
+- **Must** use dividers and groups to separate logical clusters of navigation items.
+- **Must not** use vertical navigation with a header and a sub navigation menu. Having too many places for navigation will be confusing for users.
+
+## Nav Icons
+
+- **Must** maintain consistency across using icons on navigation items. Avoid using icons to some of top navigation items and not to others.
```

Comment:
```suggestion
- Design teams **must** maintain consistency in using icons on navigation items. Avoid using icons for some top navigation items while leaving others without, as this can create confusion and disrupt the user's navigation experience. Consistent icon usage enhances predictability and clarity, ensuring all users can effectively navigate the interface.
```